File: python/1041.py
import sys
e = list(map(float, input().split(' ')))
x = e[0]
y = e[1]

if x == 0 and y!=0:
    print("Eixo Y")
    sys.exit()
elif y == 0 and x != 0:
    print("Eixo X")
    sys.exit()
if x > 0 and y > 0:
    print("Q1")
elif x < 0 and y > 0:
    print("Q2")
elif x < 0 and y < 0:
    print("Q3")
elif x > 0 and y < 0:
    print("Q4")
else:
    print("Origem")
    

File: js/PROBLEMS/1041.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split(' ')

x = parseFloat(lines[0])
y = parseFloat(lines[1])


if(x === 0 && y!=0){
    console.log("Eixo Y");
    return;
}

else if(y === 0 && x != 0){
    console.log("Eixo X");
    return;
}
if( x > 0 && y > 0){
    console.log("Q1");
}
else if( x < 0 && y > 0){
    console.log("Q2");
}
else if( x < 0 && y < 0){
    console.log("Q3");
}
else if( x > 0 && y < 0){
    console.log("Q4");
}
else{
    console.log("Origem");
}